easybuild/easyconfigs/c/CGAL/CGAL-4.11.3-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,37 @@
+name = 'CGAL'
+version = '4.11.3'
```

Comment:
Any reason to stick with the 4.11 release? There are releases up to 4.14, so I think we should go for one of
```
    CGAL 4.13.1 (April 2019)
    CGAL 4.12.2 (April 2019)
    CGAL 4.14 (March 2019)
```
https://www.cgal.org/releases.html

Reply:
There was a change in build process between 4.11 and 4.12. I'm afraid that 4.12 would require also a change in easyblock which is too complicated task for me. At least simply upgrading the library version fails during sanity check.

Reply:
+1 on makes the necessary changes to the CGAL easyblock to support latest, looks like sticking would result in lagging behind a bit too much...

@furstj Do you have more details on how the sanity check failed exactly?

Reply:
Looking into this

Reply:
The fix for this is in the easyblock is already merged, see https://github.com/easybuilders/easybuild-easyblocks/pull/1679

Reply:
In that case: @furstj please update to latest version of `CGAL`, and use `cgal.py` from https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/c/cgal.py (which you can grab via 'Raw' and supply to `eb` via `--include-easyblocks`)

Reply:
@Micket , your comments are resolved in latest commit - CGAL is updated to v4.14 (thanks to akesandgren).